litestar/contrib/piccolo.py

```diff
@@ -21,12 +21,22 @@
 from piccolo.table import Table
 
 from litestar.typing import FieldDefinition
+from litestar.utils import warn_deprecation
 
 T = TypeVar("T", bound=Table)
 
 __all__ = ("PiccoloDTO",)
 
 
+warn_deprecation(
+    deprecated_name="litestar.contrib.piccolo",
+    version="2.3.2",
+    kind="import",
+    removal_in="3.0",
+    info="importing from 'litestar.contrib.piccolo' is deprecated, please import from 'litestar.plugins.piccolo' instead",
+)
+
+
```

Comment:
can `PiccoloDTO` be imported from `litestar.plugins.piccolo` so we don't have to duplicate the rest of the module?